it/config.json

```diff
@@ -245,7 +245,7 @@
       },
       {
         "Language": "python",
-        "Rationale": "https://github.com/microsoft/kiota/issues/5637"
+        "Rationale": "https://github.com/microsoft/kiota/issues/5744"
```

Comment:
If we merge https://github.com/microsoft/kiota/pull/5724 first, I'll push a commit here to remove this entry as @shemogumbe is OOF.